pkg/operator/deploy/deploy.go

```diff
@@ -314,6 +319,23 @@ func (o *operator) IsReady(ctx context.Context) (bool, error) {
 	return true, nil
 }
 
+// o.oc.Properties.IngressProfile
+func GetDefaultImageIP(ingressProfiles []api.IngressProfile) (string, error) {
+	if ingressProfiles == nil || len(ingressProfiles) < 1 {
+		return "", fmt.Errorf("No Ingress Profiles found")
+	}
+	ingressIP := ingressProfiles[0].IP
+	if len(ingressProfiles) > 1 {
+		for _, p := range ingressProfiles {
+			if p.Name == "default" {
+				ingressIP = p.IP
+				return ingressIP, nil
```

Comment:
nit: we can skip assignment and return `p.IP`
```suggestion
				return p.IP, nil
```

Reply:
Good point, thanks!

Reply:
fixed, thank you!